packages/superset-ui-preset-chart-xy/src/components/ChartFrame.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { isDefined } from '@superset-ui/core';
+
+function checkNumber(input: any): input is number {
+  return isDefined(input) && typeof input === 'number';
+}
+
+interface Props {
+  contentWidth?: number;
+  contentHeight?: number;
+  height: number;
+  renderContent: ({ height, width }: { height: number; width: number }) => React.ReactElement;
+  width: number;
+}
+
+export default class ChartFrame extends React.PureComponent<Props, {}> {
+  static defaultProps = {
+    renderContent() {},
+  };
+
+  render() {
+    const { contentWidth, contentHeight, width, height, renderContent } = this.props;
+
+    const overflowX = checkNumber(contentWidth) && contentWidth > width;
+    const overflowY = checkNumber(contentHeight) && contentHeight > height;
+
+    if (overflowX || overflowY) {
+      return (
+        <div
+          style={{
+            height,
+            overflowX: overflowX ? 'scroll' : 'hidden',
```

Comment:
sorry this got through in the prev component, `overflow` should never be `scroll` it should always be `auto`. `scroll` always shows a scrollbar whereas `auto` does not.